src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -17,7 +17,7 @@
 public class ManufacturerDaoImpl implements ManufacturerDao {
     @Override
     public Manufacturer create(Manufacturer manufacturer) {
-        String query = "INSERT INTO manufacturers (name, country) "
+        String query = "INSERT INTO manufacturers (manufacturer_name, manufacturer_country) "
```

Comment:
it is not recommended to have the name of the table in column so better leave name and country instead of manufacturer_name, manufacturer_country. Same thing everywhere in your column names